inc/saitypes.h

```diff
@@ -211,6 +211,10 @@ typedef struct _sai_u8_list_t {
     uint8_t *list;
 } sai_u8_list_t;
 
+/**
+ * @brief Defines a s8 list or string.
+ * string should be null terminated and count should include '\0'    
```

Comment:
@lguohan - Based on our discussion we thought of adding the \0 also in count right. Otherwise we can leave it to strlen behavior.